_posts/2018-06-23-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Something old, something new…"
+date: 2018-06-23 16:00:00
+categories: preview
+tags: HackIllinois2018, HackIllinois2019, featured
+image: /assets/article_images/mission-statement-2019/landing-page-2019.jpg
+author: Kyle Begovich
+---
+
+Even though it is only June,  planning for HackIllinois 2019 is already well underway, and our Summer Team would love to offer you an inside look at some of our motivations and guiding principles.
+
+
+# Something Old:
+
+The blog has already explored a few retrospective pieces of the event last year, so here we will begin focusing on how it all applies to next year. We are currently in the process of engaging mentors from last year to get their take on their weekend on campus: what they liked and what they would like changed. These mentors are at the heart of our hacker experience, tons of fun to host, and exceptionally talented contributors in their field. If you didn’t get to engage with the mentors at the event, we hope to make this opportunity more central to the hacker experience in the future.
+
+2018 also saw some very engaging workshops and lightning talks, teaching everything from Unity development to making memes. Our sponsors and students always do a great job in offering supplemental knowledge and tools that you may not otherwise have had the opportunity to experience. Our sponsors are some of the biggest, best, and most cutting-edge in the tech industry, and the value they provide to the event and the attendees cannot be overstated. Being able to take a break from your project is something all of our attendees should feel comfortable doing, and we’ll continue to develop these events for the years to come.
```

Comment:
cutting-edge -> cutting-edge players
Otherwise, the sentence is implicitly:
Our sponsors are some of the biggest, best, and most cutting-edge sponsors in the tech industry